analytic_base_department/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2011-2018 Camptocamp SA
+# Copyright 2013 Daniel Reis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Base Analytic Department Categorization",
```

Comment:
@Saran440 It would be great if you can add summary about the module !

Reply:
Okay I will. Thank you for your reviewed. :)

Reply:
@MeetKD I added it. Please, Could you review and approved this?